tests/tests/test_sun_earth/test_sun_earth.py

```diff
@@ -65,11 +65,11 @@ def _planck(lam, teff):
 
 def sun_earth_result():
     pb_str = 'Bolometric:900-40000'
-    mypb = phoebe.atmospheres.passbands.get_passband(pb_str)
+    mypb = phoebe.get_passband(pb_str)
 
     # theoretical result: Planck x passband, no limb-darkening
     sedptf = lambda w: _planck(w, c.T_sun.si.value)*mypb.ptf(w)
-    sb_flux = np.pi*integrate.quad(sedptf, mypb.ptf_table['wl'][0], mypb.ptf_table['wl'][-1])[0] # Stefan-Boltzmann flux
+    sb_flux = np.pi*integrate.quad(sedptf, mypb.ptf_table['wl'][0], mypb.ptf_table['wl'][-1])[0]  # Stefan-Boltzmann flux
```

Comment:
would be nice to put into release-2.5 directly, but not a huge deal since this would be a fairly straightforward conflict to resolve.